themes/grass/layouts/support/list.html

```diff
@@ -0,0 +1,54 @@
+{{ partial "head.html" . }}
+{{ partial "banner.html" . }}
+
+{{ "<!-- navigation -->" | safeHTML }}
+<header class="shadow-bottom position-relative">
+  <div class="container">
+    {{ partial "navigation.html" . }}
+  </div>
+</header>
+{{ "<!-- /navigation -->" | safeHTML }}
+
+
+    <section>
+      <section class="pt-4 bg-gray">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 text-center">
+              <h2 class="section-title">Find community and commercial support for GRASS GIS</h2>
+            </div>
+          </div>
+        </div>
+	      <div class="container">
+          <div class="row">
+            <div class="col-12 bg-white mb-4">
+              The Geographic Resources Analysis Support System (<a href="https://grass.osgeo.org/" target="_blank">https://grass.osgeo.org/</a>),
+              commonly referred to as GRASS GIS, is an Open Source Geographic Information System providing powerful raster, vector and geospatial
+              processing capabilities. It can be used either as a stand-alone application or as backend for other software packages such as 
+              <a href="https://qgis.org" target="_blank">QGIS</a> and <a href="https://www.r-project.org/" target="_blank">R</a>
+              or in the cloud. It is distributed freely under the terms of the GNU General Public License (GPL).
+              GRASS GIS is a founding member of the Open Source Geospatial Foundation (<a href="https://www.osgeo.org" target="_blank">OSGeo</a>).
+            </div>
+          </div>
+	      </div>
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-12">
+	      {{ range where .Pages "File.Dir" "in" "/support/" }}   
+              <div class="section bg-white mb-4">
+                <div class="row nm">
+                  <div class="col-1 text-center"><i class="fa fa-file-text-o fa-3x grey-color-light"></i></div>
+                  <div class="col-11">
+		    <h3 class="mt-2 mb-2"><a href="{{.File.TranslationBaseName}}">{{ .Title }}</a></h3>
+                  </div>
+                </div>
+		</div>
+	       {{ end }}  
+            </div>
+          </div>
+        </div>
+      </section>{{ partial "footer.html" . }}
```

Comment:
weirdly footer.html has a closing </body></html> which turns this into invalid html, an option would be to move it at the end of the file

Reply:
mhm... I didn't change anything of that, just copied the layouts as they were and made minimal changes